cmd/prow-job-dispatcher/README.md

```diff
@@ -0,0 +1,19 @@
+# prow-job-dispatch
+
+As designed in [DPTP-1152](https://issues.redhat.com/browse/DPTP-1152) this tool chooses a cluster in the CI build farm for Prow jobs.
```

Comment:
Please copy the design from the ticket here, someone may delete the ticket in the future (I have found references to non-existing tickets before) or not anyone might have permissions to view it

Reply:
Done.